src/code/gui.cpp

```diff
@@ -645,4 +645,18 @@ void Gui::renderTextBar() {
 
     SDL_RenderFillRect(renderer, &rect2);
 
+}
+
+void Gui::renderFreeSpace(){
+    SDL_Texture *textTex;
+    SDL_Rect textRec;
+    SDL_Rect rect;
+    rect.x = atoi(themeData.values["fsposx"].c_str());
```

Comment:
can you add a default values here in case theme.ini does not have anything, as it will render in 0,0 ? just to support already made themes.

Reply:
I do not mean our two themes ;)  ... I mean themes created by other people with theme.ini compatible with 0.5 and earlier